assignment01/assignment1.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function($){
+    $('#first-name').html('Jen');
+    $('#last-name').html('Cink');
+    $('.mcc').css({'color': 'blue', 'font-style': 'italic'});
+    $('#hobby').append('<li>Reading</li>', '<li>Cooking</li>');
+});
```

Comment:
great anonymous function! nice job targeting those content and style changes by id with jquery!